src/posts/2021-09-03-getting-started-with-orca.md

```diff
@@ -0,0 +1,80 @@
+- Type `orca` on a terminal window and then press the <kbd>Return</kbd> key. Alternatively, you can type `orca` in the Run dialog and press the <kbd>Return</kbd> key. You can use the key combination <kbd>Alt</kbd> + <kbd>F2</kbd> to open the Run dialog and the <kbd>ESC</kbd> key to close it. If you launch Orca using the Run dialog, you can close it by typing the command `pkill orca` or `killall orca` on the terminal or in the the Run dialog and then pressing the <kbd>Return</kbd> key.
+- Open the Settings dialog and then select Universal access. You can then toggle the screen reader on or off.
+
+After launching Orca, it will immediately start narrating the content of the focused application.
+
+## The Orca Modifier keys
+
+The modifier key used by Orca depends on the keyboard layout. Orca has two keyboard layouts, namely:
+
+- Desktop keyboard layout
+- Laptop keyboard layout
+
+The Desktop keyboard layout is the default keyboard layout. Its Modifier key is the key combination <kbd>Insert</kbd> + <kbd>KP_Insert</kbd>. <kbd>KP_Insert</kbd> is the same as <kbd>KP_0</kbd>. If it is your first time using Orca, most likely you are in the Desktop keyboard layout.
+
+The modifier key for the Laptop layout is <kbd>CapsLock</kbd> .
+
+You can change the keyboard layout from the preferences section. You can open the preferences dialog from the terminal or from the Run dialog by typing the command `orca -s` or `orca --setup` and then pressing the <kbd>Return</kbd> key. That command will open Orca and the preferences dialog at the same time.
+
+## The Learn Mode
+
+There is a feature in Orca referred to as Learn Mode. It is a valuable feature if you are still learning Orca. It helps you to access the documentation quickly. If you press any key in this mode, Orca will announce the pressed keystroke and the associated Orca command bound to it.
+
+Below is a table showing the keyboard shortcuts associated with the Learn Mode. You need to first turn on Learn mode using the key combination <kbd>orca modifier</kbd> + <kbd>H</kbd> before using the commands starting in the second row.
+
+| Command                                 | Description                                                    |
+| --------------------------------------- | -------------------------------------------------------------- |
+| <kbd>orca modifier</kbd> + <kbd>H</kbd> | Turn on Learn Mode                                             |
+| <kbd>F1</kbd>                           | Open Orca documentation                                        |
+| <kbd>F2</kbd>                           | Open shortcuts that apply _orca-wide_                          |
+| <kbd>F3</kbd>                           | Open shortcuts that apply to the currently focused application |
+| <kbd>Up</kbd> or <kbd>Down</kbd>        | Navigate open list of shortcuts                                |
+| <kbd>ESC</kbd>                          | Turn off Learn Mode                                            |
```

Comment:
I really appreciate the use of tables and the `kbd` element to help the reader out 🙂